### PR TITLE
Updated .gitignore

### PR DESCRIPTION
**Pull request by PerikiyoXD**

### CODE REVIEW

**Comment by PerikiyoXD** [comment]
Fail

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,36 @@
+# Ignore certain files #
+
+# CMake - General Files #
+
+cmake/
```
Comment (danhedron) [inline .gitignore]: There's already a directory in the repository called ``cmake/``, what is this ignoring?

File: .gitignore [hunk 2]
```diff
@@ -1,7 +1,43 @@
...
+install_manifest.txt
+
+# CMake - Visual Studio 14 2015 - Files #
+
+*.suo
+*.sln
+*.vcxproj*
+*.VC*
+
+*.a
```
Comment (danhedron) [inline .gitignore]: Shouldn't this be under "Build"?

**Comment by PerikiyoXD** [comment]
+ Added some Visual studio extensions

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # OpenRW
 
-[![Build Status](https://travis-ci.org/rwengine/openrw.svg?branch=master)](https://travis-ci.org/rwengine/openrw)
+[![Build Status](https://travis-ci.org/PerikiyoXD/openrw.svg?branch=master)](https://travis-ci.org/PerikiyoXD/openrw)
```
Comment (JayFoxRox) [inline README.md]: Unintended change?

**Comment by PerikiyoXD** [comment]
Bump

**Comment by danhedron** [comment]
What's with the updates to inih and mman-win32?

**Pull request closed without merging**